src/clr-angular/button/button-loading/loading-button.ts

```diff
@@ -73,12 +73,14 @@ export class ClrLoadingButton implements LoadingListener {
     switch (state) {
       case ClrLoadingState.DEFAULT:
         this.renderer.removeStyle(this.el.nativeElement, 'width');
+        this.renderer.removeStyle(this.el.nativeElement, 'transform');
         if (!this.disabled) {
           this.renderer.removeAttribute(this.el.nativeElement, 'disabled');
         }
         break;
       case ClrLoadingState.LOADING:
         this.setExplicitButtonWidth();
+        this.renderer.setStyle(this.el.nativeElement, 'transform', 'translatez(0)');
```

Comment:
Can we apply these styles to the animation specs or through the host binding instead of using Renderer2? Is that possible? Renderer2 ought to be used in cases where we need to directly manipulate DOM styles without dealing with Angular change detection.

Reply:
Cory asked me this question, it's a matter of synrhonicity to interact with animations, and leftover attributes.
For instance, you can't do `this.renderer.removeAttribute(this.el.nativeElement, 'disabled')` with `@HostBinding`. Any value you use would leave a disabled attribute on the element no matter what.
And for the synchronicity, remember that HostBinding doesn't do anything until the view step of change detection, which can be problematic for our integration with animations.

I think this is 👌.

Reply:
Assigning`null` or `undefined` to the value can remove the `disabled` attribute though 😛 

I am just a bit concerned we might adopt a habit of resorting to doing things manually controlling DOM attributes/styles while there is an more "Angular way" option.

But this fix is fine by me as we had already been using Renderer2 in the component.